src/index.ts

```diff
@@ -74,7 +81,8 @@ async function serverChannel(
             started = true;
           }
         },
-      } as any,
+        // as any due to CLI mistyping: https://github.com/chromaui/chromatic-cli/pull/800
+      } as Partial<Parameters<typeof run>[0]["options"]> as any,
```

Comment:
I think this can be updated now that the fix is released in `chromatic@6.23.1`.

Reply:
Will do after merging